src/SixLabors.Fonts/FontCollection.cs

```diff
@@ -15,13 +16,24 @@ namespace SixLabors.Fonts
     /// </summary>
     public sealed class FontCollection : IFontCollection
     {
-        private readonly Dictionary<string, List<IFontInstance>> instances = new Dictionary<string, List<IFontInstance>>(StringComparer.OrdinalIgnoreCase);
-        private readonly Dictionary<string, FontFamily> families = new Dictionary<string, FontFamily>(StringComparer.OrdinalIgnoreCase);
+        private readonly Dictionary<string, List<IFontInstance>> instances = new Dictionary<string, List<IFontInstance>>(StringComparer.CurrentCultureIgnoreCase);
+        private readonly Dictionary<string, FontFamily> families = new Dictionary<string, FontFamily>(StringComparer.CurrentCultureIgnoreCase);
+        private readonly CultureInfo culture;
```

Comment:
And if LCID is supported (for example in .net standard 2.0+), then we could also add a `Dictionary<int, Dictionary<string, FontFamily>> familiesByLanguages`  where `int` is an `LCID`, to add every font family name in every language. These entries are already inside `NameTable` so I think probably it's better to make font families names available to the public instead of keeping them private whenever possible. Each value of `familiesByLanguages` (i.e. each `Dictionary<string, FontFamily>`) uses its own `StringComparer` in its own culture.

When a search begins, firstly we search in the preferred culture (i.e. `FontCollection.families`). If we didn't find anything,  (and if LCID is supported, i.e. we are .net standard 2.0+ and we have the above `FontCollection.familiesByLanguages`), then we iterate through `FontCollection.familiesByLanguages` and search for the font family in every possible language.

Reply:
decided to go with make users specify culture when searching they want to use culture specific names